openforcefield/utils/toolkits.py

```diff
@@ -727,7 +727,6 @@ def describe_oeatom(oeatom):
                 raise UndefinedStereochemistryError(msg)
             else:
                 print(msg)
-                return
```

Comment:
Per our discussion, I'm ok with returning the Molecule in these cases, but let's make sure to comeback and add a hefty warning for users who disable this check.

Reply:
Sounds great!